app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/supportingevidence/do_you_want_to_upload.scala.html

```diff
@@ -61,7 +61,8 @@
                 hasErrors = hasErrors,
                 selected = form.value,
                 helpText = Some(Html(messages(s"$key.help-text"))),
-                yesContent = None
+                yesContent = None,
+                warningField= Some(messages(s"$key.hint"))
             )
```

Comment:
We need to display this warning message only if user selects "Yes" in repayment page.